beamer/chain.py

```diff
@@ -228,22 +228,29 @@ def _transact(func: ContractFunction, **kwargs: Any) -> HexBytes:
     return txn_hash
 
 
+def process_l1_resolutions(context: Context) -> None:
+    log.info("Processing L1 resolutions", num_tasks=len(context.l1_resolutions))
```

Comment:
We should use `debug` here, both because in other similar places we use `debug` and also because it's going to get called every second or so.